src/LandIce/evaluators/LandIce_BasalMeltRate_Def.hpp

```diff
@@ -16,38 +16,51 @@
 namespace LandIce
 {
 
+template<typename Type>
```

Comment:
Could you move this to a common header, like `Albany_Utils.hpp` or one of those?

Reply:
Woops! This got left in from a previous commit. These functions have already been put in a new file called Albany_KokkosUtils.hpp that was included with the Volume PR.